chain/chain.go

```diff
@@ -25,7 +25,7 @@ type Chain struct {
 	Name      string   `json:"name"`
 	Genesis   *Genesis `json:"genesis"`
 	Params    *Params  `json:"params"`
-	Bootnodes []string `json:"bootnodes"`
+	Bootnodes []string `json:"bootnodes,omitempty"`
```

Comment:
Why do you need the `omitempty` here if this field can never be unpopulated?

Reply:
This field is unpopulated in some e2e test case scenarios.

Reply:
Why would we put `omitempty` because of that?

We should never modify the actual implementation to conform to some tests in this sense, but the other way around